lib/matplotlib/tests/test_axes.py

```diff
@@ -2064,6 +2065,24 @@ def test_errorbar():
 
     fig.suptitle('Variable errorbars')
 
+@cleanup
+def test_errorbar_shape():
+    fig = plt.figure()
+    ax = fig.gca()
+    
+    x = np.arange(0.1, 4, 0.5)
+    y = np.exp(-x)
+    yerr1 = 0.1 + 0.2*np.sqrt(x)
+    yerr = np.vstack((yerr1, 2*yerr1)).T
+    xerr = 0.1 + yerr
+    
+    print(np.shape(y))
+    print(np.shape(yerr))
```

Comment:
Tiny issue but I would suggest removing these prints  

They will show up in the test output with -s passed to nose without any context